src/components/DateRangePickerInput.jsx

```diff
@@ -252,14 +263,15 @@ function DateRangePickerInput({
         openDirection={openDirection}
         onChange={onEndDateChange}
         onFocus={onEndDateFocus}
-        onKeyDownTab={onEndDateTab}
         onKeyDownArrowDown={onKeyDownArrowDown}
         onKeyDownQuestionMark={onKeyDownQuestionMark}
         verticalSpacing={verticalSpacing}
         small={small}
         regular={regular}
       />
 
+      {datePickerIsAfterEndInput && children}
```

Comment:
I think this can be `{isEndDateFocused && children}`

Reply:
🎉